### ISSUE TITLE
Tabindex/What should be tabbable

### ISSUE THREAD

**Issue by tspeed90** [fac-13/research#4]
When working on our project, I wasn't exactly clear on what should be accessible by tabbing and I went a bit overboard on using tabindex. Now that I had some time to look into it, it seems that tabindex should only be used on elements that are interactive. Things that do something when you click on them (buttons, links, check boxes, etc). I used it on bodies of text and on our profile cards, but this was an incorrect use of the tabindex attribute.
